BookClub/BookClub.xproj

```diff
@@ -15,5 +15,9 @@
   <PropertyGroup>
```

Comment:
Let's not modify the main /Bookclub project. Create another one called `BookClub.Web` or similar name